runelite-client/src/main/java/net/runelite/client/plugins/itemidentification/ItemIdentification.java

```diff
@@ -69,6 +69,49 @@
 	SITO_FOIL(Type.HERB, "Sito", "SF", ItemID.SITO_FOIL, ItemID.GRIMY_SITO_FOIL),
 	SNAKE_WEED(Type.HERB, "Snake", "SW", ItemID.SNAKE_WEED, ItemID.GRIMY_SNAKE_WEED),
 	VOLENCIA_MOSS(Type.HERB, "Volenc", "V", ItemID.VOLENCIA_MOSS, ItemID.GRIMY_VOLENCIA_MOSS),
+	
+	
+	//Logs
+	RED_LOGS(Type.LOGS, "Red", "RED", ItemID.RED_LOGS),
+	GREEN_LOGS(Type.LOGS, "Green", "GRE", ItemID.GREEN_LOGS),
+	BLUE_LOGS(Type.LOGS, "Blue", "BLU", ItemID.BLUE_LOGS),
+	WHITE_LOGS(Type.LOGS, "White", "WHI", ItemID.WHITE_LOGS),
+	PURPLE_LOGS(Type.LOGS, "Purple", "PUR", ItemID.PURPLE_LOGS),
+
+	SCRAPEY_TREE_LOGS(Type.LOGS, "Scrapey", "SCRAP", ItemID.SCRAPEY_TREE_LOGS),
+
+	LOG(Type.LOGS, "Log", "LOG", ItemID.LOGS),
+	ACHEY_TREE_LOG(Type.LOGS, "Achey", "ACH", ItemID.ACHEY_TREE_LOGS),
+	OAK_LOG(Type.LOGS, "Oak", "OAK", ItemID.OAK_LOGS),
+	WILLOW_LOG(Type.LOGS, "Willow", "WIL", ItemID.WILLOW_LOGS),
+	TEAK_LOG(Type.LOGS, "Teak", "TEAK", ItemID.TEAK_LOGS),
+	JUNIPER_LOG(Type.LOGS, "Juniper", "JUN", ItemID.JUNIPER_LOGS),
+	MAPLE_LOG(Type.LOGS, "Maple", "MAP", ItemID.MAPLE_LOGS),
+	MAHOGANY_LOG(Type.LOGS, "Mahogany", "MAH", ItemID.MAHOGANY_LOGS),
+	ARCTIC_PINE_LOG(Type.LOGS, "Arctic", "ARC", ItemID.ARCTIC_PINE_LOGS),
+	YEW_LOG(Type.LOGS, "Yew", "YEW", ItemID.YEW_LOGS),
+	BLISTERWOOD_LOG(Type.LOGS, "Blist", "BLI", ItemID.BLISTERWOOD_LOGS),
+	MAGIC_LOG(Type.LOGS, "Magic", "MAG", ItemID.MAGIC_LOGS),
+	REDWOOD_LOG(Type.LOGS, "Red", "RED", ItemID.REDWOOD_LOGS),
+
+	PYRE_LOGS(Type.LOGS, "Pyre", "P", ItemID.PYRE_LOGS),
+	ARCTIC_PYRE_LOGS(Type.LOGS, "Artic  P", "AP", ItemID.ARCTIC_PYRE_LOGS),
+	OAK_PYRE_LOGS(Type.LOGS, "Oak P", "OAKP", ItemID.OAK_PYRE_LOGS),
+	WILLOW_PYRE_LOGS(Type.LOGS, "Willow P", "WILP", ItemID.WILLOW_PYRE_LOGS),
+	TEAK_PYRE_LOGS(Type.LOGS, "Teak P", "TEAKP", ItemID.TEAK_PYRE_LOGS),
+	MAPLE_PYRE_LOGS(Type.LOGS, "Maple P", "MAPP", ItemID.MAPLE_PYRE_LOGS),
+	MAHOGANY_PYRE_LOGS(Type.LOGS, "Mahogany P", "MAHP", ItemID.MAHOGANY_PYRE_LOGS),
```

Comment:
Does this overlap the item next to it? "Mahogany P" seems pretty long

Reply:
Yes this did overlap. found a few more and also fixed those